source/_sass/lib/site.sass

```diff
@@ -63,25 +63,42 @@ header.masthead
 
 /* nav */
 
+#mainNav
+  @media screen and (max-width: 768px)
+    position: absolute
+    .container
+      margin-right: 0
+      margin-left: 0
+      padding-left: 0
+      padding-right: 0
+      position: relative
+      z-index: 9999
+    .nav
+      display: list-item
+    .navbar-collapse
+      width: 100%
+      margin-left: -15px
+      li a
+        text-indent: 20px
+
 #mainNav .container
-  float: left
   margin: 0 auto
-
-  @media screen and (max-width: 768px)
-    float: right
-    margin-right: 0px
-    margin-left: 0px
-    padding-left: 0px
-    padding-right: 0px
-    position: relative
-    z-index: 9999
-
-  .nav
+  .navbar-toggle
+    padding: 15px 10px
+  .navbar-brand
+    padding: 0 40px 0 0
     @media screen and (max-width: 768px)
-      display: list-item
+      padding: 0 40px 15px 15px
+    > img
```

Comment:
is above a typo?

Reply:
No, I wrote that on purpose. In this case, CSS is only valid up to one level and not at deeper levels. This can be useful if you want to add an additional image next to the logo in a different size.

Example

```html
        <a class="navbar-brand" href="/">
          <img id="logo" alt="oVirt" src="{{ site['logo'] }}">
          <span><img id="another-pics" [...] /></span>
        </a>
```

If this is not necessary, I simply remove it or just refer to #logo.

Reply:
thanks for explanation